build/windows/installer/conf/fluent-bit-common.conf

```diff
@@ -56,3 +56,12 @@
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
+
+# TODO: Merge this with oms_output plugin as both of them are using the same plugin
```

Comment:
same here .. are we merging with with oms_output plugin? or are we going with separate? if its latter, lets remove the TODOs.

Reply:
removed the TODOs....Talked to Amol and decided that we need to keep this separate